### PR TITLE
Match on going UI effort to complete order

### PR DESCRIPTION
**Pull request by HaroldFS**
This PR addresses the following:

Update to CheckoutTableViewCell UI + lite documentation

- Added lite documentation to CheckoutTableViewCell file
- Added lite documentation to CheckoutTableViewController for proper sectioning
- updated the UI for the CheckoutTableViewCell

Add checkbox button + logic

- added logic to the CheckoutTableViewController
- added nib file for checkbox

### CODE REVIEW

File: swift/ios-shoppe-demo/Views/CheckBoxButtonCell.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import Foundation
+import UIKit
+
+class CheckBoxButtonCell: UITableViewCell {
+    @IBOutlet weak var checkboxButton: UIButton!
+    var boxIsChecked: Bool = false
+    var checkoutView: CheckoutTableViewController?
+
+    @IBAction func checkbox(_ sender: Any) {
+        boxIsChecked = boxIsChecked ? false : true
```
Comment (sabrina-li) [inline swift/ios-shoppe-demo/Views/CheckBoxButtonCell.swift]: nit: maybe `boxIsChecked = !boxIsChecked` would be more readable?

Comment (HaroldFS) [reply swift/ios-shoppe-demo/Views/CheckBoxButtonCell.swift]: I agree :)

File: swift/ios-shoppe-demo/Views/CheckBoxButtonCell.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
+//
+//  CheckBoxButtonCell.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 5/28/20.
+//  Copyright © 2020 Harold Davis Jr. All rights reserved.
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Views/CheckBoxButtonCell.swift]: Personal copyright

File: swift/ios-shoppe-demo/Views/CheckoutTableViewController.swift [hunk 3]
```diff
@@ -74,18 +105,15 @@ class CheckoutTableViewController: UITableViewController {
                 cell?.setup()
             }
 
+            return cell ?? UITableViewCell()
+        case 5:
+            let cell =  tableView.dequeueReusableCell(withIdentifier: "checkoutBox") as? CheckBoxButtonCell
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Views/CheckoutTableViewController.swift:110]: nit: double space; does iOS have a linter or format on save?

**Review by van-fs — APPROVED** [review]
LGTM, see my two comments prior to merge

File: swift/ios-shoppe-demo/Views/CheckoutTableViewController.swift (hunk 3 above)
Comment (HaroldFS) [reply swift/ios-shoppe-demo/Views/CheckoutTableViewController.swift:110]: Im doing research into a linter :)

**Pull request merged**